lib/utils/logger/index.js

```diff
@@ -168,6 +168,7 @@ function logRequest(message, params) {
   const instance = Logger.getInstance();
 
   instance.output.push([timeIso, lodashEscape(message), lodashEscape(inspectObject(params))]);
+  instance.commandOutput.push([timeIso, lodashEscape(message), lodashEscape(inspectObject(params))]);
```

Comment:
`output` is the same as `commandOutput`

Reply:
Both `output` and `commandOutput` have `httpOutput`. But, `output` stores the `httpOutput` for the entire testsuite, while `commandOutput` only holds the `httpOutput` for a single command and is reset once used. 

Although I believe this approach can be improved, but we should refactor it in a separate PR.